.gitignore

```
@@ -83,3 +83,6 @@ lcov.info
 dev-tools/iota-private-network/data
 dev-tools/iota-private-network/configs/validators/validator*
 dev-tools/iota-private-network/configs/genesis/genesis.blob
+
+# generated-documentation
+docs/generated-docs/**
```

I love new lines at the end :trollface: 
```suggestion
docs/generated-docs/**

```